### PR TITLE
i18n(pt-BR): Add `routing.md` translation

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

PT-BR translation for `routing.md`

### CODE REVIEW

File: src/pages/pt-BR/core-concepts/routing.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Roteamento
+description: Uma introdução a roteamento com Astro.
+i18nReady: true
+---
+
+Astro utiliza **roteamento baseado em arquivos** para gerar suas URLs na construção com base no layout dos arquivos no diretório `src/pages` do seu projeto. Quando um arquivo é adicionado ao diretório `src/pages` do seu projeto, ele é automaticamente disponível como uma rota baseada no seu nome de arquivo.
```
Comment (victor0x16) [inline src/pages/pt-BR/core-concepts/routing.md:8]:
```suggestion
Astro utiliza **roteamento baseado em arquivos** para gerar suas URLs na construção com base no layout dos arquivos no diretório `src/pages` do seu projeto. Quando um arquivo é adicionado ao diretório `src/pages` do seu projeto, ele é automaticamente disponibilizado como uma rota baseada no seu nome de arquivo.
```

File: src/pages/pt-BR/core-concepts/routing.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+Rotas podem ser geradas a partir de múltiplos parâmetros nomeados, a qualquer nível do caminho de arquivo:
+
+- `pages/blog/[slug].astro` → (`/blog/ola-mundo`, `/blog/postagem-2`, etc.)
+- `pages/[nomeusuario]/configuracoes.astro` → (`/fred/configuracoes`, `/drew/configuracoes`, etc.)
+- `pages/[lingua]-[versao]/info.astro` → (`/en-v1/info`, `/fr-v2/info`, etc.)
+
+#### O objeto `Astro.params`
+
+Componentes Astro que geram rotas dinamicamente tem acesso ao objeto `Astro.params` para cada rota. Isso te permite utilizar as partes geradas de uma URL em seu script e template do componente.ate.
```
Comment (victor0x16) [inline src/pages/pt-BR/core-concepts/routing.md:68]:
```suggestion
Componentes Astro que geram rotas dinamicamente tem acesso ao objeto `Astro.params` para cada rota. Isso te permite utilizar as partes geradas de uma URL em seu script e template do componente.
```

File: src/pages/pt-BR/core-concepts/routing.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+Paginação é útil quando você precisa gerar múltiplas páginas numeradas de um conjunto de dados maior:
+
+- `/postagens/1` (Página 1: Mostra as postagens 1-10)
+- `/postagens/2` (Página 2: Mostra as postagens 11-20)
+- `/postagens/3` (Página 3: Mostra as postagens 21-30)
+
+
+### A prop `page`
+
+Quando você utiliza a função `paginate()`, cada página na coleção passará seus dados através da prop `page`. A prop `page` tem diversas propriedades úteis, mas a mais importante é `page.data`. Esta é o array contendo os pedaços de dados da página que você passou para a função `paginate()`.
```
Comment (victor0x16) [inline src/pages/pt-BR/core-concepts/routing.md:181]:
```suggestion
Quando você utiliza a função `paginate()`, cada página na coleção passará seus dados através da prop `page`. A prop `page` tem diversas propriedades úteis, mas a mais importante é `page.data`. Esse é o array contendo os pedaços de dados da página que você passou para a função `paginate()`.
```

File: src/pages/pt-BR/core-concepts/routing.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+	/** número total de resultados */
+	total: number;
+	/** número da página atual, começando por 1 */
+	currentPage: number;
+	/** número de itens por página (padrão: 25) */
+	size: number;
+	/** número da última página */
+	lastPage: number;
+	url: {
+		/** url da página atuaç */
```
Comment (victor0x16) [inline src/pages/pt-BR/core-concepts/routing.md:216]:
```suggestion
		/** url da página atual*/
```

File: src/i18n/pt-BR/nav.ts [hunk 5]
```diff
@@ -19,4 +19,6 @@ export default NavDictionary({
 	'guides/imports': 'Assets Estáticos',
 	'guides/integrations-guide': 'Integrações',
 	'guides/server-side-rendering': 'Renderização no lado do Servidor (SSR)',
+	reference: 'Referência',
+	'core-concepts/routing': 'Regras de Roteamento',
```
Comment (victor0x16) [inline src/i18n/pt-BR/nav.ts:23]: Routing has been moved.

**Review by victor0x16 — CHANGES_REQUESTED** [review]

**Comment by victor0x16** [comment]
Just tiny typos!

File based routing makes static website so much more easier to build

**Comment by Yan-Thomas** [comment]
I will take this opportunity and update the recent changes as well! Thanks for your review @victor0x16!

**Review by victor0x16 — APPROVED** [review]

**Pull request merged**